### ISSUE TITLE
NewSelectEntry dropdown disabled color

### ISSUE THREAD

**Issue by xines** [fyne-io/fyne#2166]
**widget.NewSelectEntry** dropdown once disabled looks just like a dark box while using dark theme.

Disabled
![GitHub Logo](https://i.imgur.com/g07aAkg.png)

Not disabled for reference
![GitHub Logo](https://i.imgur.com/r3bhC1M.png)

Wouldnt it be quite more aesthetic to have the **not disabled** type (no button box) but grayed out icon style?

As these are just looks, i understand people can have other opinions, what you think?

**Comment by Jacalz** [comment]
It looks like you have found a bug with the SelectEntry. Do you think that you could use the issue templates (the bug template in this case) in the future please? It makes it a lot easier for us to triage and review all the issues. Thanks for understanding 🙂

**Comment by xines** [comment]
> It looks like you have found a bug with the SelectEntry. Do you think that you could use the issue templates (the bug template in this case) in the future please? It makes it a lot easier for us to triage and review all the issues. Thanks for understanding 🙂

- Yeah no problem, i was not sure what section to put this as i'm not sure if it was a bug. 😄

**Comment by Jacalz** [comment]
Thanks. Will have a look later today if I can get this fixed.

**Comment by Jacalz** [comment]
I have been looking into this. It does not look like an issue with SelectEntry, but rather an issue with how the LowImportance button is rendered when disabled. It seems to work for the light theme, but the dark theme produces that strange rectangle around it.

I have a partial fix by changing the following code to the snippet pasted below:
https://github.com/fyne-io/fyne/blob/a25b08480d742f016ee24c1b37aa02c3bb433bd5/widget/button.go#L313-L314
``go
case r.button.Disabled():
		if r.button.Importance == LowImportance {
			return theme.ButtonColor()
		}
		return theme.DisabledButtonColor()
```
This does still work fyne for the light theme, but the actual icon is now gone in dark theme. That's unfortunately as far as I managed to come with my little investigation.

**Comment by andydotxyz** [comment]
Is it possible that the issue is that the SelectEntry background is not changing correctly when disabled?

**Comment by Jacalz** [comment]
I don't think so. The select entry (the entry part of it) looks about the same as disabling a regular entry.

**Comment by andydotxyz** [comment]
Yes, but I think a disabled entry does not do what it used to...?
Anyhow, I suspect that this is not a solution that LowImportance alone can solve, we are embedding the button on a different background, so in this instance (and not with other LowImportance buttons) we want to use a transparent background.
If you apply this to all LowImportance then it will impact toolbar and others where the disabled style should include a background shift.
